microcloud/service/microovn.go

```diff
@@ -54,7 +55,30 @@ func (s OVNService) Client() (*client.Client, error) {
 
 // Bootstrap bootstraps the MicroOVN daemon on the default port.
 func (s OVNService) Bootstrap() error {
-	return s.m.NewCluster(s.name, util.CanonicalNetworkAddress(s.address, s.port), 2*time.Minute)
+	err := s.m.NewCluster(s.name, util.CanonicalNetworkAddress(s.address, s.port), 2*time.Minute)
+	if err != nil {
+		return err
+	}
+
+	ctx, cancel := context.WithTimeout(context.Background(), 1 * time.Minute)
+  defer cancel()
+	for {
+		select {
+		case <-ctx.Done():
+			return fmt.Errorf("Timed out waiting for MicroOVN cluster to initialize")
+		default:
+			names, err := s.ClusterMembers()
+			if err != nil {
+				return err
+			}
+
+			if len(names) > 0 {
+				return nil
+			}
+
+			time.Sleep(300 * time.Millisecond)
```

Comment:
Why 300ms BTW?

Reply:
Not too fast, not too slow, do you prefer something else usually?

Reply:
It depends for me. If I'm waiting for up to minute or more I tend to do 1s wait. If I'm waiting say up to 5s I would do between 100ms and 500ms.

Reply:
Fair enough, bumped it to a second.